### PR TITLE
[Docs] Databricks 

### LINKED ISSUE #7066: [DOCS] Connect Databricks Cluster to Yugabyte YSQL with Yugabyte Encryption In-Transit Enabled
@jeffjewett put together this summary of connecting Databricks Cluster to Yugabyte YSQL with Yugabyte Encryption In-Transit Enabled

This will live in /latest/develop/ecosystem-integrations/

INTERNAL DRAFT: https://docs.google.com/document/d/1B0EPALMq0hIVC9OCPtvMFWoHXERU_XvPDCFOsvbaspY/edit

CC: @nmalladi

### PR DESCRIPTION
**Pull request by polarweasel**
Resolves #7066

Add docs on how to connect Databricks to Yugabyte over TLS.

[PREVIEW is here](https://deploy-preview-7819--infallible-bardeen-164bc9.netlify.app/latest/develop/ecosystem-integrations/)

### CODE REVIEW

File: docs/content/latest/develop/ecosystem-integrations/_index.html [hunk 1]
```diff
@@ -81,19 +81,31 @@
...
-        Spring Framework support for YugabyteDB.
+        Spring Framework support for YugabyteDB
+      </div>
+    </a>
+  </div>
+
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="databricks/">
+      <div class="head">
+        <img class="icon" src="/images/section_icons/develop/ecosystem/databricks.svg" aria-hidden="true" />
```
Comment (nchandrappa) [inline docs/content/latest/develop/ecosystem-integrations/_index.html:104]: Databricks logo is not getting displayed correctly in the preview. Probably needs an image of higher resolution.

**Review by nchandrappa — APPROVED** [review]
LGTM. One small review comment on the Databricks logo.

File: docs/content/latest/develop/ecosystem-integrations/_index.html (hunk 1 above)
Comment (polarweasel) [reply docs/content/latest/develop/ecosystem-integrations/_index.html:104]: Odd, it's an SVG, so it's sharp no matter the resolution I try it at. Is the issue that it's too small?

**Review by jeffjewett — APPROVED** [review]

**Comment by polarweasel** [comment]
Closing this as it's no longer relevant

**Pull request closed without merging**